api/v1beta1/safemode.go

```diff
@@ -12,6 +12,7 @@ type UnsafemodeSpec struct {
 	DisableCountTooLarge       bool    `json:"disableCountTooLarge,omitempty"`
 	DisableNeitherHostNorPort  bool    `json:"disableNeitherHostNorPort,omitempty"`
 	DisableSpecificContainDisk bool    `json:"disableSpecificContainDisk,omitempty"`
+	AllowRootDiskFailure       bool    `json:"allowRootDiskFailure,omitempty"`
```

Comment:
This was renamed here (awesome!) but not changed yet in the docs or examples

Reply:
Oups 🙈